tests/integration/commands/deploy/deploy.test.ts

```diff
@@ -178,11 +178,13 @@ describe.skipIf(process.env.NETLIFY_TEST_DISABLE_LIVE === 'true').concurrent('co
         .withNetlifyToml({
           config: {
             build: { publish: 'public', command: 'echo "no op"' },
-            edge_functions: [{ function: 'edge', path: '/*' }],
```

Comment:
this was making this test pass, but also it meant that
```
'should deploy Edge Functions when directory exists'
```
test description was not correct, as with no explicit edge functions config, edge functions were NOT deployed